pkgs/servers/nosql/mongodb/mongodb.nix

```diff
@@ -11,6 +11,17 @@ with stdenv.lib;
 }@args:
 
 let
+  ifdef = if versionAtLeast version "4.2"
+    then { python = python38.withPackages (ps: with ps; [ pyyaml cheetah3 psutil setuptools ]);
+            scons = scons;
```

Comment:
Maybe inherit? I do like the symmetry with the one below though :)

Reply:
this reads a bit clearer to me, though admittedly i am a novice here.